### ISSUE TITLE
Accessibility audit: (JAWS) Nunjucks Tab missing from list item list

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#299]
## Describe the issue
Global issue to all views with examples. 
The Nunjucks tab on every example is missing within JAWS list item list. It shouldn't be.
 
## Steps to reproduce the issue
With JAWS on navigate to a view with an example and invoke the list item list. 
 
## Actual vs expected behaviour
This shouldn't be included in this list. The Nunjucks list item has a missing presentation role attribute.

## Fix Recommendation
- [x] Add 'role='presentation' to  the list item with a class of "app-tabs__list-item".

## Related stories
- [Accessibility audit: (JAWS) Tab component missing expanded states](https://github.com/dwp/design-system/issues/300)
- [Accessibility audit: (JAWS) Tab component, tab differentiation with hidden text when using multiple examples](https://github.com/dwp/design-system/issues/301)
- [Accessibility audit: (JAWS) Tabs appear as links but missing for link items list](https://github.com/dwp/design-system/issues/302)
- [Accessibility audit: Tabs UI different from GDS Tab component](https://github.com/dwp/design-system/issues/303)
